core/engine/engine_test.go

```diff
@@ -0,0 +1,7 @@
+package engine
+
+import "testing"
+
+func TestHelloWorld(t *testing.T) {
```

Comment:
ℹ️ remove or implement

Reply:
It's a placeholder to remind me that I need to implement tests, as I don't think I'll have time to jump into it today, that is why many things are in a "work in progress" state. I wanted to give others visibility early to iterate on the implementation.